ui/packages/catalog-ui-search/src/main/webapp/js/model/User.js

```diff
@@ -213,7 +213,7 @@ User.Preferences = Backbone.AssociatedModel.extend({
   },
   savePreferences: function() {
     const currentPrefs = this.toJSON()
-    if (_.isEqual(currentPrefs, this.lastSaved)) {
+    if (_.isEqual(Common.duplicate(currentPrefs), this.lastSaved)) {
```

Comment:
This has already been added to the original PR which is yet to be merged: https://github.com/codice/ddf/pull/4466/files